### ISSUE TITLE
Write an exercise that attendees should be able to complete after the course

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/machine-learning-novice-sklearn#4]
This will help determine what the whole lesson is trying to get people to be able to achieve.

**Comment by mgalland** [comment]
Suggestions:

1. Attendees should be able to split a dataset into a train + a validation + a test set.
2. Attendees should be able to choose the right algorithm based on characteristics of their dataset and their output (categories, continuous). Something like: https://scikit-learn.org/stable/tutorial/machine_learning_map/index.html
3. Attendees should be able to perform a grid search to search for the best algorithm parameters.
4. Attendees should be able to import various datasets in Python preferably (?) using pandas. 

😃 please expand 😃
